configure.ac

```diff
@@ -643,7 +643,7 @@ case $host in
 
          bdb_prefix=$($BREW --prefix berkeley-db4 2>/dev/null)
```

Comment:
I guess it shouldn't even *call* brew here in case bdb is not used?

Reply:
brew just returns the expected directory (based on the available - also uninstalled – formulas). But we could also just avoid the brew call (complicated the folding though...)